src/index.html

```diff
@@ -22,6 +22,404 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="header__link"
+      >
+        <img
+          src="/src/images/logo.png"
+          alt="logo"
+          class="logo"
+        />
+      </a>
+      <nav class="header__nav">
```

Comment:
Using the header prefix, you bind the navigation block to this header.
BEM idea that you can reuse blocks.
For example, let’s just imagine that you have exactly the same navigation in your footer, but different footer styles
In this case, you will have to duplicate the code and write a separate navigation for the footer
Therefore, it is better to separate the navigation from the header so that it is a separate block
```suggestion
      <nav class="nav">
```